roles/monitoring_sap/tasks/repos-activation.yml

```diff
@@ -0,0 +1,17 @@
+---
+# The role common will activate SLES OS Repos and install the prometheus packages
+
+- name: Install netcat-openbsd
+  ansible.builtin.zypper:
+    name: netcat-openbsd
+    state: present
+  tags: netcat-openbsd
+
+- name: Activate SLES repo SLE-Module-SAP-Applications15-SP5-Updates & SLE-Module-Packagehub-Subpackages15-SP5-Pool
+  ansible.builtin.command: "{{ item_cmd }}"
+  loop:
+    - hostname
+    - /usr/sbin/SUSEConnect -p sle-module-sap-applications/15.5/x86_64
+    - /usr/sbin/SUSEConnect -p PackageHub/15.5/x86_64
```

Comment:
15.5 should not be hardcoded. Should be obtained from the system dynamically based on OS

Reply:
What if version 15.6 , 15.7 comes ? Will you again hard code it ? what about then support for 15.5 ?

Reply:
I confirm, we have to check every version and release manually and adapt Ansible which version will be allowed.
Possibly we stay we remain on OS-Version even if a newer is available. Most likely Andreas and Arnold will decide.
I wish we could use any version.

Reply:
changed to  {{ ansible_distribution_version }}